### PR TITLE
Organized the list of Fragments in Alphabetical order

### PR DESCRIPTION
**Pull request by jaimehta19**
## Explanation

Arranges the list of fragments in alphabetical order in FragmentComponent.kt.

## Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by jaimehta19** [comment]
@rt4914 @BenHenning @FareesHussain

**Comment by rt4914** [comment]
@jaimehta1912 Can you please pick issues from [here](https://github.com/oppia/oppia-android/issues?q=is%3Aopen+is%3Aissue+no%3Aassignee+label%3A%22good+first+issue%22) ?

**Comment by rt4914** [comment]
@jaimehta1912 Can you please pick issues from [here](https://github.com/oppia/oppia-android/issues?q=is%3Aopen+is%3Aissue+no%3Aassignee+label%3A%22good+first+issue%22) ?

File: app/src/main/java/org/oppia/android/app/fragment/FragmentComponent.kt [hunk 1]
```diff
@@ -7,11 +7,6 @@ import org.oppia.android.app.administratorcontrols.AdministratorControlsFragment
 import org.oppia.android.app.administratorcontrols.appversion.AppVersionFragment
 import org.oppia.android.app.completedstorylist.CompletedStoryListFragment
 import org.oppia.android.app.deprecation.AutomaticAppDeprecationNoticeDialogFragment
-import org.oppia.android.app.devoptions.DeveloperOptionsFragment
-import org.oppia.android.app.devoptions.markchapterscompleted.MarkChaptersCompletedFragment
-import org.oppia.android.app.devoptions.markstoriescompleted.MarkStoriesCompletedFragment
-import org.oppia.android.app.devoptions.marktopicscompleted.MarkTopicsCompletedFragment
-import org.oppia.android.app.devoptions.vieweventlogs.ViewEventLogsFragment
```
Comment (BenHenning) [inline app/src/main/java/org/oppia/android/app/fragment/FragmentComponent.kt:14]: Why are these being removed?

**Review by BenHenning — CHANGES_REQUESTED** [review]
Thanks @jaimehta1912. I have one main question, but otherwise I suggest picking a starter issue as suggested by @rt4914 above. I'm not opposed to accepting changes like this, but our issue tracker has higher priority tasks to complete on it.

**Pull request closed without merging**